scripts/core/editor3/actions/spellchecker.jsx

```diff
@@ -26,3 +26,17 @@ export function refreshWord(word) {
         payload: word
     };
 }
+
+/**
+ * @ngdoc method
+ * @name setAutoSpellchecker
+ * @param {Boolean} auto True if the autospellchecker is enabled
```

Comment:
Can you please change the name of this parameter to `enabled`? It makes more sense.
Also, it might be more correct to say: `True, if automatic spellchecking should be enabled`.